OracleWebCenterPortal/COPYRIGHT.txt

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2020, Oracle Corporation.  All rights reserved.
```

Comment:
Please remove this file. We already have a `LICENSE.txt` in the root of the repository.